CONTRIBUTING.md

```diff
@@ -30,20 +39,34 @@ First of all, thank you for contributing to Meilisearch! The goal of this docume
 
 ## Development Workflow
 
+### Requirements
+
+- [Composer 2.x](https://getcomposer.org/download/)
+
 ### Setup
 
+Clone the project then:
+
 ```bash
+cd meilisearch-php
 composer install
 ```
 
+If you don't already have a Meilisearch server instance running, execute the following commands:
+
+NB: Will create a bunch of files and folders, better to use a dedicated folder outside of this project.
```

Comment:
This will not create a bunch of files, only the directory `data.ms` (database files) whose path you can easily configure with an environment variable or a flag

Reply:
You're right, just did read https://docs.meilisearch.com/learn/configuration/instance_options.html carefully.

I'll remove this part.